src/libstd/fs.rs

```diff
@@ -1299,6 +1299,20 @@ impl DirBuilder {
 
     /// Create the specified directory with the options configured in this
     /// builder.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// #![feature(dir_builder)]
+    /// use std::path::Path;
+    ///
+    /// let path = Path::new("/tmp/foo/bar/baz");
+    /// std::fs::DirBuilder::new()
+    ///     .recursive(true)
+    ///     .create(path).unwrap();
+    ///
+    /// assert!(std::fs::metadata(path).unwrap().is_dir());
```

Comment:
Conventionally these sorts of functions aren't referenced via `std` at the top-level, so you may want to add an import to the top of the example:

```rust
use std::fs::{self, DirBuilder};

DirBuilder::new() ...

fs::metadata(path) ...
```